nixos/modules/services/monitoring/prometheus/exporters/nats.nix

```
@@ -0,0 +1,34 @@
+{ config, lib, pkgs, options }:
```

```suggestion
{ config, lib, pkgs, options, ... }:
```

modules need to leave room for unspecified options as you never know when new ones could be added